sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryServices.java

```diff
@@ -117,19 +117,18 @@ JobStatistics dryRunQuery(String projectId, JobConfigurationQuery queryConfig)
   interface DatasetService {
     /**
      * Gets the specified {@link Table} resource by table ID.
+     *
+     * <p>Returns {@code null} if no such table exists.
      */
     Table getTable(String projectId, String datasetId, String tableId)
         throws InterruptedException, IOException;
 
     /**
-     * Returns a table, or {@code null} if no such table exists.
-     */
-    Table getTable(TableReference table) throws InterruptedException, IOException;
```

Comment:
(note from offline chat)
apply this change to your PR, and verify existing getTable() calls.